sympy/plotting/plot.py

```diff
@@ -1559,21 +1560,23 @@ def plot(*args, **kwargs):
     for arg in plot_expr:
         expr = arg[0]
         limit = arg[1]
-        frac = fraction(expr) #to find the denominator
-        new_deno = frac[1]
-        if (not new_deno.is_Integer) and (not new_deno.is_Float) and (not new_deno.is_irrational):
-            root_deno = real_roots(new_deno)   #to find the points of discontinuity
+        root_deno = []
+        for exp in denoms(expr):
```

Comment:
It would be nice to use `d` or `den` instead of `exp` which looks like SymPy function `exp()`

Reply:
I'll replace this